gpii/node_modules/preferencesServer/src/preferencesService.js

```diff
@@ -16,22 +16,58 @@ var fluid = fluid || require("infusion");
 (function () {
     "use strict";
 
-    var gpii = fluid.registerNamespace("gpii");
+    var gpii = fluid.registerNamespace("gpii"),
+        $ = fluid.registerNamespace("jQuery");
+    require("gpii-express-user");
 
     fluid.defaults("gpii.preferencesServer.preferencesService", {
         gradeNames: ["fluid.component"],
         components: {
             dataStore: {
                 type: "gpii.dbOperation.dataStore"
+            },
+            expressUserUtils: {
+                type: "gpii.express.user.utils",
```

Comment:
I am very suspicious of a component named "utils" ...... what does it do?

Reply:
This is a component in the `gpii-express-user` module that contains the routines for creating and unlocking users.

https://github.com/GPII/gpii-express-user/blob/010bbaf83ae2e43558878f77b43e13c60223a0e3/src/js/server/utils.js#L29